notebook/static/notebook/less/cell.less

```diff
@@ -1,3 +1,8 @@
+._marked_style(@n) {
+    border-left-width: @n;
+    padding-left: 6px-@n;
```

Comment:
One small tiny nitpicking I'm not sure about:  The 6px here should likely be a variable used somewhere else, when cells are both not marked and not selected. 

Likely because line 11 `border-width:thin` == `1px` and line 49: padding is 5px :-P